logstash-core/lib/logstash/instrument/metric_type/counter.rb

```diff
@@ -22,7 +22,7 @@ def decrement(value = 1)
       @counter.decrement(value)
     end
 
-    def execute(type, key, action, time, value)
+    def execute(namespace, key, type, action, value)
```

Comment:
This is a generic call, I wasnt sure it was needed I just fixed the signature of the method.
I need to revisit that.